conda/plugins/types.py

```diff
@@ -141,3 +141,12 @@ class CondaAuthHandler(NamedTuple):
 
     name: str
     handler: type[ChannelAuthBase]
+
+
+class CondaHealthChecks(NamedTuple):
+    """
+    Return type to use when defining conda health checks plugin hook.
+    """
+
+    name: str
+    action: Callable(str)
```

Comment:
```suggestion
    action: Callable[[], None]
```

Type hinting uses square brackets. The type hint above means, "any callable object that accepts no arguments and returns None".